README.md

```diff
@@ -1,5 +1,17 @@
 # trace-server-protocol
 
+The Trace Server Protocol is a protocol designed to allow frontend visualizers to be decoupled from backend. It allows the frontend to query resources in a RESTful way. The guiding principles:
```

Comment:
Not sure about the RESTful way,
- as TSP currently bundles a lot of request payload into the body.
- Recent cross-team discussions also highlighted the current limitations in terms of resource/entity modeling, starting at the request URI level.
- We also mentioned the domain language which could improve from an ubiquity perspective.
- All in all, this README may mention the REST/http basis of TSP, without claiming REST-fullness, I think.